src/Microsoft.Health.Fhir.Core/Features/Search/SearchValues/ISupportSortSearchValue.cs

```diff
@@ -17,12 +17,12 @@ public interface ISupportSortSearchValue : IRangedComparable
         /// Determines whether this current value is the minimum when compared to
         /// a collection of search values for the same parameter.
         /// </summary>
-        bool IsMin { get; set; }
+        bool? IsMin { get; set; }
```

Comment:
>bool? [](start = 8, length = 5)

Can I ask what does null value means?
Right now it just answer question: is this value are minimal value or not, and there is yes or no answer. What would null add to that statement?

Reply:
Not set, if the search parameter is not supported (i.e. by config or the database) then the work to figure out min/max is skipped. We then cannot trust that it is the minimum which is the default

Reply:
>it is the minimum which is the default.
Default for bool is false, so I assume you talking about code which set this values. Can we modify that code rather than expand this concept?


---
In reply to: [585764243](https://github.com/microsoft/fhir-server/pull/1647#discussion_r585764243) [](ancestors = 585764243)